pitch.md

```diff
@@ -1,44 +1,80 @@
-experience like now? What would their experience be like if you could help 
-them?_
+_What need is met by your idea? Who are you helping? What is that person's experience like now? What would their experience be like if you could help them?_
 
+Shared spaces are often managed by some form of scheduler but the interfaces we use for them are nonintuitive and rather poor. This could help anyone who shares a resource with others, such as MuddVans, conference rooms, and study rooms. One example is a HMC student trying to reserve the Green Room. The student has to submit reservation details before being able to see the schedule. Then, they can only see 12 hours intervals of the schedule and cannot alter their original reservation without creating an entirely new query. This makes deciding on dates and times for meetings more difficult than it should be. If I could help them, the user would be able to query schedules in different ways (e.g. "Show me the schedule on November 28th for rooms that can hold 20 people"; "Reserve the Green room from 8PM to 9PM on November 28th"). There would also be ways to describe a new space that can also be reserved ("New Blue Room in Platt").
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+I think that similar to SQL, this task is better suited as a query system. Although, the most intuitive reservation system may be a visual, a query system can be the internal DSL for a future GUI project. For calendar users, it will provide a smoother experience than the one currently found on EMS.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+When I try to organize events, it is fairly frustrating to check multiple dates and times or try to do multiple reservations. When I was trying to reserve a MuddVan, I came up with the idea to try something similar and it also aligned with my proposal in free-1.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain? Be careful
 to distinguish between, e.g,. a graphical interface and a linguistic interface._ 
 
+The user would be able to do three basic tasks.
+* Create spaces that can be reserved
+* Reserve spaces for periods of times
+* Query for space availability
+
+The user interacts through text queries. The programming will look similar to other query languages like SQL. 
+
+* Create spaces
+
+This would ideally return a confirmation or an error.
+
+```
+CREATE ROOM "Blue Room" IN "Platt"
+```
+
+* Reserve spaces
+
+This would ideally return either a confirmation or an error.
+
+```
+RESERVE ROOM "Blue Room" FROM "1600" TO "2030" ON "11/28/2015"
+```
+
+* Query for availability
+
+This would ideally return a list of reservations on that day, or a list of available times.
+
+```
+CHECK ROOMS IN "Platt" ON "11/28/2015"
+```
```

Comment:
Another potentially useful feature would be the ability to cancel reservations as a user or as an administrator.